generate-stackbrew-library.sh

```diff
@@ -111,8 +111,6 @@ for version; do
 
 		fullVersion="$(jq -r '.[env.version].version' versions.json)"
 
-		[[ "$fullVersion" == *.*[^0-9]* ]] || fullVersion+='.0'
```

Comment:
This change is technically valid even without this PR, but it's currently harmless (however with it in this PR, we end up with tags like `tip-YYYYMMDD.0`, which is undesirable).